src/index.ts

```diff
@@ -208,17 +208,53 @@ export class StorageProvider {
     }
   };
 
-  myFiles = async () => {
+  numOfMyFiles = async () => {
```

Comment:
You can fetch total number of files using the gateway api. 

Refer this [PR](https://github.com/arcana-network/gateway/pull/80)